src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -599,6 +599,18 @@ struct
              ~f:(fun sum account ->
                sum + Currency.Balance.to_int account.balance ) )) ;
     Coda_metrics.(Counter.inc_one Transition_frontier.root_transitions) ;
+    let consensus_state = Breadcrumb.consensus_state new_root in
+    let blockchain_length =
+      consensus_state |> Consensus.Data.Consensus_state.blockchain_length
+      |> Coda_numbers.Length.to_int |> Float.of_int
+    in
+    let global_slot =
+      consensus_state |> Consensus.Data.Consensus_state.global_slot
+      |> Float.of_int
+    in
+    Coda_metrics.(
+      Gauge.set Transition_frontier.slot_fill_rate
+        (blockchain_length /. global_slot)) ;
```

Comment:
Fine to land this as is, but I think we should be tracking a "global" slot fill rate (which this is) and a "local" slot fill rate (as in, what is the slot fill rate over some window of slots, like 8K or 24K slots or something).

Reply:
Let's land and then we can fix in a followup